deny.toml

```
@@ -31,6 +31,7 @@ version = 2
 ignore = [
   "RUSTSEC-2024-0384", # Waiting for https://github.com/console-rs/indicatif/pull/666
   "RUSTSEC-2024-0436", # https://rustsec.org/advisories/RUSTSEC-2024-0436 - paste is unmaintained - https://github.com/dtolnay/paste
+  "RUSTSEC-2025-0014", # Waiting for https://github.com/apache/arrow-rs/issues/7264
```

```suggestion
  "RUSTSEC-2025-0014", # TODO(#9255): Waiting for https://github.com/apache/arrow-rs/issues/7264
```

That way our stale-issue script can pick it up